content/dstu2/authorization.md

```diff
@@ -14,7 +14,7 @@ In order for your client application to utilize any protected resources, your cl
 * Name - Application name, this should match what the name of the app will be in the app store  
 * Redirect/Callback URI  
 * Email address  
-* Logo URI - The logo will need to be a [Scalable Vector Graphics][6] image
+* Logo URI - The logo will need to be a [Scalable Vector Graphics][6] image  
```

Comment:
From what I can tell looking at the site locally, the double spaces at the end here don't do anything, same for L39